src/partials/cart.hbs

```diff
@@ -0,0 +1,63 @@
+<div class="wrapper">
+  <!--Login Section-->
+  <div class="cart-section">
+    <div class="cart_header">
+      <h3>My Cart</h3>
+      <span>(1 item)</span>
+    </div>
+    <div class="cart_contents">
+      <table class="cart_contents-table">
```

Comment:
Why do we need a table? You can create similar layout using CSS or use CSS flex/grid layout wherever required.

Reply:
removed